openquake/qa_tests_data/event_based/mutex/expected/ruptures.csv

```diff
@@ -1,8 +1,8 @@
-#,,,,,,,,,"generated_by='OpenQuake engine 3.13.0-git1e2939e5df', start_date='2022-02-01T17:49:35', checksum=2517070861, investigation_time=50.0, ses_per_logic_tree_path=2000"
+#,,,,,,,,,"generated_by='OpenQuake engine 3.13.0-gitd378a898c3', start_date='2022-07-13T18:56:26', checksum=2517070861, investigation_time=50.0, ses_per_logic_tree_path=2000"
 rup_id,multiplicity,mag,centroid_lon,centroid_lat,centroid_depth,trt,strike,dip,rake
-0,1,8.2,140.002,35.234,1.94E+01,Subduction Interface,214.6394,42.8531,9.00E+01
-1,3,8.3,140.859,35.099,1.55E+01,Subduction Interface,221.0664,46.28668,9.00E+01
-2,8,7.9,140.555,35.076,1.58E+01,Subduction Interface,220.6537,43.46952,9.00E+01
-3,10,7.9,139.616,35.413,1.85E+01,Subduction Interface,207.4639,42.44744,9.00E+01
-4,15,8,139.588,35.324,1.58E+01,Subduction Interface,207.5819,43.06566,9.00E+01
-5,3,8.2,140.859,35.301,1.92E+01,Subduction Interface,222.5578,46.38767,9.00E+01
\ No newline at end of file
+0,1,8.20E+00,140.002,35.234,1.94E+01,Subduction Interface,128.7383,42.8531,9.00E+01

```

Comment:
Now I am confused. This test shows that we were already computing the strike and dip. I understood you wanted to add the get_strike and get_dip methods because they were not supported. Were these methods/values wrong? Also, I would like to see a simpler test that shows that the methods you are adding work.

Reply:
the simpler tests are here https://github.com/gem/oq-engine/blob/d378a898c3cbe2b10cfeb1b7718ffc19b4eafc9f/openquake/hazardlib/tests/geo/surface/gridded_test.py#L73

The file `openquake/qa_tests_data/event_based/mutex/expected/ruptures.csv` on master contains `nan` in [strike and dip](https://github.com/gem/oq-engine/blob/master/openquake/qa_tests_data/event_based/mutex/expected/ruptures.csv) columns. Hence, I used the [source model files](https://github.com/gem/oq-engine/blob/master/openquake/qa_tests_data/event_based/mutex/source_model.xml) containing the coordinates for gridded surface and calculated the respective strike and dip as per the current implementation. 

during the previous commits to  `get_strike` and `get_dip`, I had calculated the same as per the previous implementation. Hence, in this commit, these column values have changed as per the new implementation.